### PR TITLE
Wrong branch checked out in docker image setup page in Core docs - Closes #634

### LINKED ISSUE #634: Wrong branch is checked out on Docker image setup page
**Describe the bug**
In `setup/docker.adoc` of the `docs-core` branch, it is described to check out the `master` branch of the `lisk-core` repository, but instead the tag `v3.0.0-beta.0` should be checked out.

### PR DESCRIPTION
**Pull request by chris529**
Closes issue #634

### CODE REVIEW

File: modules/ROOT/pages/setup/docker.adoc [hunk 1]
```diff
@@ -120,13 +120,13 @@ Clone the {url_lisk_core_repository}[Lisk Core Repository^].
...
 cd lisk-core/docker                                <3>
-git checkout master                                <4>
+git checkout v3.0.0-beta.0                         <4>
 ----
 
 <1> Switch to lisk user.
 <2> Clone the repository.
 <3> Navigate into the docker directory.
-<4> Checkout master branch for last stable version of Lisk Core.
+<4> Checkout version 3.0.0-beta.0 for the latest stable version of Lisk Core.
```
Comment (Tschakki) [inline modules/ROOT/pages/setup/docker.adoc]:
```suggestion
<4> Checkout the `3.0.0-beta.0` branch, to install Lisk Core version 3.0.0.
```

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/setup/docker.adoc (hunk 1 above)
Comment (chris529) [reply modules/ROOT/pages/setup/docker.adoc]: Yes, I like it Mona! 👍

**Review by Tschakki — APPROVED** [review]
Thank you! :)

**Review by chris529 — COMMENTED** [review]
Yes, I like it Mona! 👍

**Pull request merged**